source/MonoGame.Extended/Screens/Transitions/ExpandTransition.cs

```diff
@@ -34,7 +34,7 @@ public override void Draw(GameTime gameTime)
             var height = _graphicsDevice.Viewport.Height * Value;
             var rectangle = new RectangleF(x, y, width, height);
 
-            _spriteBatch.Begin(samplerState: SamplerState.PointClamp);
+            _spriteBatch.Begin(SpriteSortMode.Deferred, null, samplerState: SamplerState.PointClamp, null, null);
```

Comment:
Not possible to add an extension for .Begin(), because of the similarity in the parameters with the existing one. 
We have to go with XNA compatible code.